pkg/cache/cache_test.go

```diff
@@ -402,7 +402,7 @@ var validFS = fstest.MapFS{
 	"name": "clusterwide-alpha",
 	"entries": [
 		{"name": "etcdoperator.v0.9.0"},
-		{"name": "etcdoperator.v0.9.2-clusterwide", "replaces": "etcdoperator.v0.9.0", "skips": ["etcdoperator.v0.6.1","etcdoperator.v0.9.0"], "skipRange": ">=0.9.0 <=0.9.1"},
```

Comment:
Is this change related to the model validation change somehow? It seems unrelated to me at first glance.

Reply:
Checked the catalogs impact and it looks the same as before.  HOWEVER, we no longer need this test change, because somehow it's OK for 0.9.2-clusterwide to skip AND replace v0.9.0 ...?